docs/plugins/plugin-writer/first-plugin.rst

```diff
@@ -1,2 +1,109 @@
 Writing Your First Plugin
 =========================
+
+For a complete list of what should be done to have a working plugin, check :doc:`checklist`.
+In this section key parts of plugin implementation are covered in more detail to help you as a plugin writer to get started.
+
+Understanding models
+--------------------
+There are models which are expected to be used in plugin implementation, so understanding what they are designed for is useful for a plugin writer.
```

Comment:
Would it be possible to limit the lines to 100 chars and wrap the lines? We don't have a docs style guide, but it prevents the code from wrapping when its read on github. If not, we can fix it afterwards too.

Reply:
I can do it for the line like you pointed here but I'm not sure how to wrap lines which are part of list (aka start with `*`). If I try to do so, lists are no longer rendered correctly.

Reply:
I think you indent the second line to start underneath the first character of the bulleted line. Like this?

```
* this is a really long line
  that won't end
```